doc/source_names.tex

```diff
@@ -28,33 +28,35 @@ \section*{Names}
+\item \texttt{\href{https://sicp.comp.nus.edu.sg/chapters/17}{runtime()}}: Returns number of milliseconds elapsed since January 1, 1970 00:00:00 UTC
 \item \verb#parse_int#\texttt{(s, i)}:
 interprets the \emph{string} \texttt{s} as an integer, using the positive integer \texttt{i} as radix, and returns the respective value,
 see \href{https://www.ecma-international.org/ecma-262/9.0/index.html#sec-parseint-string-radix}{\color{DarkBlue}ECMAScript Specification, Section 18.2.5}.
-\item \verb#undefined#, \verb#NaN#, \verb#Infinity#: Refer to JavaScript's undefined,
+\item \href{https://sicp.comp.nus.edu.sg/chapters/4}{\texttt{undefined}},
+  \texttt{\href{https://www.ecma-international.org/ecma-262/9.0/index.html\#sec-value-properties-of-the-global-object-nan}{\color{DarkBlue}NaN}}, \texttt{\href{https://www.ecma-international.org/ecma-262/9.0/index.html\#sec-value-properties-of-the-global-object-infinity}{\color{DarkBlue}Infinity}}: Refer to JavaScript's undefined,
 NaN (``Not a Number'') and Infinity values, respectively.
-\item \verb#is_boolean#\texttt{(x)}, \verb#is_number#\texttt{(x)}, \verb#is_string#\texttt{(x)}, \verb#is_function#\texttt{(x)}:
+\item \verb#is_boolean#\texttt{(x)}, \href{https://sicp.comp.nus.edu.sg/chapters/36}{\lstinline{is_number(x)}},
+  \href{https://sicp.comp.nus.edu.sg/chapters/36}{\lstinline{is_string(x)}}, \verb#is_function#\texttt{(x)}:
         return \texttt{true} if the type of \texttt{x} matches the function name and \texttt{false} if it does not. Following
         JavaScript, we specify that \verb#is_number# returns \texttt{true} for \texttt{NaN} and \texttt{Infinity}.
 \item \texttt{prompt(s)}: Pops up a window that displays the \emph{string} \texttt{s}, provides
 an input line for the user to enter a text, a ``Cancel'' button and an ``OK'' button. The call of \texttt{prompt}
 suspends execution of the program until one of the two buttons is pressed. If 
 the ``OK'' button is pressed, \texttt{prompt} returns the entered text as a string.
 If the ``Cancel'' button is pressed, \texttt{prompt} returns a non-string value.
-\item \texttt{display(x)}: Displays the value \texttt{x} in the console\footnote{The notation used for the display of values is consistent with \href{http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf}{JSON}, but also displays \texttt{undefined} and function objects.}; returns the argument \texttt{a}.  
-\item \texttt{error(x)}: Displays the value \texttt{x} in the console\footnotemark[\value{footnote}] with error flag. The evaluation
+\item \href{https://sicp.comp.nus.edu.sg/chapters/17\#footnote-6}{\texttt{display(x)}}: Displays the value \texttt{x} in the console\footnote{The notation used for the display of values is consistent with \href{http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf}{\color{DarkBlue}JSON}, but also displays \texttt{undefined} and function objects.}; returns the argument \texttt{a}.  
```

Comment:
Nothing wrong with this link, but the text in the textbook might be slightly outdated, referring to type coercion of `+` although we now use strict type checks.